pool/app/invitation/repo/repo.ml

```diff
@@ -244,10 +244,10 @@ let bulk_insert ?mailing_id pool contacts experiment_id =
       (Dynparam.empty, [])
       invitations
   in
-  let (Dynparam.Pack (pt, pv)) = values in
   let prepare_request =
     let open Caqti_request.Infix in
-    Format.asprintf "%s \n %s" insert_sql (CCString.concat ",\n" value_insert)
+    let values = CCString.concat ",\n" value_insert in
+    [%string "%{insert_sql} \n %{values} \n ON DUPLICATE KEY UPDATE updated_at = now()"]
```

Comment:
Was that an issue before without `ON DUPLICATE KEY UPDATE updated_at = now()`?

Reply:
Happens only during testing with initial seed and then running the worker locally